src/studio-home/StudioHome.test.jsx

```diff
@@ -249,5 +259,23 @@ describe('<StudioHome />', async () => {
       expect(getByText('Looking for help with Studio?')).toBeInTheDocument();
       expect(getByText('LMS')).toHaveAttribute('href', process.env.LMS_BASE_URL);
     });
+
+    it('should show search button and open search modal when button clicked, if search flag enabled', async () => {
+      setConfig({
+        ...getConfig(),
+        MEILISEARCH_ENABLED: 'false',
+      });
+      const { queryByRole } = render(<RootWrapper />);
+      expect(queryByRole('button', { name: 'Search content' })).not.toBeInTheDocument();
+    });
+
+    it('should show search button if meilisearch flag disabled', async () => {
+      setConfig({
+        ...getConfig(),
+        MEILISEARCH_ENABLED: 'false',
+      });
+      const { queryByRole } = render(<RootWrapper />);
+      expect(queryByRole('button', { name: 'Search content' })).not.toBeInTheDocument();
+    });
```

Comment:
These 2 testcases seem to have the same test code? And it seems to be doing the opposite of what is described in the first case, and the second case should be "should _not_ show" right?

Reply:
I didn't check if the search button was showing in the header. This test was not working. I removed it here: 0d58997747fc7830424748e6edb0edd34de52b34

Thank you for pointing out!